### PR TITLE
Add Eslint 

### PR DESCRIPTION
**Pull request by ad-astra-per-ardua**
#4 Changes:

1. **ESLint Configuration**: Introduced a new ESLint configuration tailored for our project, especially for TypeScript support.
2. **Lint Script**: Added a lint script to `package.json` for ease of use.
3. **TypeScript Linting**: Installed necessary ESLint packages to support TypeScript.

Added Packages:

- eslint: Ensures consistent code styles.
- @typescript-eslint/parser: ESLint parser for TypeScript.
- @typescript-eslint/eslint-plugin: ESLint plugin containing TypeScript specific linting rules.
- (Any other ESLint related packages you've added)

### CODE REVIEW

**Comment by corners2wall** [comment]
@d982h8st7  Could you lint project? please :pleading_face:

**Comment by ad-astra-per-ardua** [comment]
> @d982h8st7 Could you lint project? please 🥺

You mean run lint on your project?

**Comment by corners2wall** [comment]
> > @d982h8st7 Could you lint project? please 🥺
> 
> You mean run lint on your project?

yeap

**Comment by corners2wall** [comment]
Closes #4

**Comment by ad-astra-per-ardua** [comment]
> > > @d982h8st7 Could you lint project? please 🥺
> > 
> > 
> > You mean run lint on your project?
> 
> yeap

image

Is it ok?

**Comment by corners2wall** [comment]
@d982h8st7  do commit. I will fix this in this branch
UPD: you can fix this if you want

**Comment by ad-astra-per-ardua** [comment]
> @d982h8st7 do commit. I will fix this in this branch UPD: you can fix this if you want

Sorry but, Could specify what files to commit? is there any prob on lint?

**Comment by corners2wall** [comment]
I forgot what it's called when you run esLint over the entire project and it fixes all your errors

Maybe [this](https://stackoverflow.com/questions/40271230/how-to-run-eslint-fix-from-npm-script) idk

**Comment by ad-astra-per-ardua** [comment]
> I forgot what it's called when you run esLint over the entire project and it fixes all your errors
> 
> Maybe [this](https://stackoverflow.com/questions/40271230/how-to-run-eslint-fix-from-npm-script) idk

Umm... is this correct? execute --fix and just commit remain file. sorry for the frustrating.

**Comment by corners2wall** [comment]
Not bad. Thanks

**Comment by corners2wall** [comment]
@d982h8st7  Could you resolve conflicts? :pleading_face:

**Comment by ad-astra-per-ardua** [comment]
> @d982h8st7 Could you resolve conflicts? 🥺

sure working on it

File: .eslintrc.cjs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  env: {
+    es6: true,
+    node: true,
+    browser: true
+  },
+  parserOptions: {
+    ecmaVersion: 'latest',
+    sourceType: 'module'
+  }
+};
```
Comment (corners2wall) [inline .eslintrc.cjs]:
```suggestion
};

```

**Comment by corners2wall** [comment]
Thanks a lot @d982h8st7 
Come again :wave:

**Comment by ad-astra-per-ardua** [comment]
> Thanks a lot @d982h8st7 Come again 👋

I learned a lot. Thank you have a good day! 👋

**Pull request merged**